plugin/editorconfig.vim

```diff
@@ -390,11 +433,6 @@ augroup END
 function! s:UseConfigFiles_Python_Builtin() " {{{2
 " Use built-in python to run the python EditorConfig core
 
-    " ignore buffers that do not have a file path associated
```

Comment:
Thank you. Now I can see your comments.

It is done in top of s:UseConfigFile (L374-L378 after this change): https://github.com/editorconfig/editorconfig-vim/pull/82/files#diff-b96ea8b60d0436a02a99512c2e5942fdR374